### ISSUE TITLE
Incorrect configuration for image in SQLCE

### ISSUE THREAD

**Issue by statler** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#564]
When generating the configuration for SqlCe image, the following is 
```c#
Property(x => x.Logo)
    .HasColumnName(@"Logo")
    .HasColumnType("image")
    .IsOptional()
    .HasMaxLength(null);
```
This throws an exception when trying to write any value to a table in which this property exists.

The following however, does work.
```c#
Property(x => x.Logo)
    .HasColumnName(@"Logo")
    .HasColumnType("image")
    .IsOptional()
    .IsMaxLength();
```

**Comment by sjh37** [comment]
Just about to jump on a plane. I'll look into this later.

**Comment by sjh37** [comment]
Is this for EF6 or .Net Core ?
Take a look at case #179 which the solution is for using `.HasMaxLength(null);` for images > 4000 bytes. 

`.IsMaxLength()` is used for `ntext` column types.

@ErikEJ Have you ever heard of `.HasMaxLength(null);` throwing an exception when trying to write any value to a table in which this property exists?

**Comment by ErikEJ** [comment]
We  need to know if this is for EF Core or EF 6.

**Comment by statler** [comment]
EF 6 thanks Erik

**Comment by statler** [comment]
BTW - this is exactly the issue listed by Listenreich in #179. Images >4000 bytes

**Comment by ErikEJ** [comment]
Yes, but #179 is EF6, not Core.

**Comment by ErikEJ** [comment]
OK, so now it is EF 6 after all??? 😕 

>> This throws an exception when trying to write any value to a table in which this property exists.
 
@statler  No, I have never heard about this, so please share the full stack trace of the exception, and your table schema.

**Comment by statler** [comment]
Yes. Ef6. I had a typo and edited it almost immediately. Did you refer only to the email? Sorry about that.

Won't be able to get the stack trace quickly. Away from my PC on leave. But it is easy to replicate, just try to map an image greater than 4kb with the default code - e.g. `HasMaxLength(null)`. You will get the error per #179

**Comment by ErikEJ** [comment]
Well, in that case the generated code should most likely jsúst be changed to be:

    .IsMaxLength();

**Comment by statler** [comment]
Yes. Completely agree.

**Comment by sjh37** [comment]
Thanks both. I'll update the code when I get back from holiday.

**Comment by sjh37** [comment]
SQL CE support was removed in v4, so closing this.
